packages/reporting/src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
```

Comment:
What do all these acronyms mean? Are all these metrics client side?

Reply:
You can check that here https://web.dev/vitals/

Reply:
Mmm... but now that I'm thinking better it shouldn't affect us much. Since this would be used locally. I'll removed from now.